app/pages/profile/comment-link.cjsx

```diff
@@ -57,13 +57,13 @@ module.exports = React.createClass
           <span>
             {' '}in{' '}
             <a href={@state.href}>
-              {if @state.project? and @state.owner
+              {if @state.project? and @state.owner? and !@props.project?
```

Comment:
yes, would prefer not refactor as part of this - don't want to add anything else into this PR.